lib/EvalSourceMapDevToolModuleTemplatePlugin.js

```diff
@@ -58,9 +58,7 @@ class EvalSourceMapDevToolModuleTemplatePlugin {
 			});
 			sourceMap.sources = moduleFilenames;
 			if(sourceMap.sourcesContent) {
```

Comment:
You can remove this whole statement, since it no longer modifies the sourcesContent